modules/runner/src/integrationTest/java/org/apache/ignite/internal/sql/api/ItSqlSynchronousApiTest.java

```diff
@@ -159,10 +167,20 @@ public void dml() {
         IgniteSql sql = igniteSql();
```

Comment:
Could you please add the following case for both Sync and Async embedded API implementation:
- start explicit transaction
- insert some rows to a table
- rollback transaction
- verify rows were not inserted

Reply:
Oh, I see there is something similar in implicitTransactionsWithDml test. Let's split this onto several smaller tests